### PR TITLE
fix frequency for SPI2/3

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/i2c.rs [hunk 1]
```diff
@@ -253,12 +253,12 @@ macro_rules! hal {
                     pins: PINS,
                     mode: Mode,
                     clocks: Clocks,
-                    apb: &mut APB1,
```
Comment (TheZoq2) [inline src/i2c.rs]: Is this a breaking change?

If I understand it correctly, it shouldn't be even though the type of a public function is changing

File: src/rcc.rs [hunk 2]
```diff
@@ -634,3 +664,8 @@ bus! {
     TIM10 => (APB2, tim10en, tim10rst),
     TIM11 => (APB2, tim11en, tim11rst),
 }
+
+#[cfg(any(feature = "stm32f102", feature = "stm32f103"))]
+bus! {
+    USB => (APB1, usben, usbrst),
```
Comment (TheZoq2) [inline src/rcc.rs]: This should probably be used by the USB code, right?

File: src/i2c.rs (hunk 1 above)
Comment (burrbull) [reply src/i2c.rs]: No it is not breaking change (it is still `APB1`).

**Comment by burrbull** [comment]
> Looks good for the most part, I added two comments in the code.
> 
> Also, would you mind adding a changelog entry. Or perhaps that's not required since most of this doesn't touch public API apart from fixing the wrong clock being used for SPI

I think it make sense to release 0.5.1 with this patch, because SPI2 and SPI3 work with incorrect frequency for now.

**Comment by TheZoq2** [comment]
> I think it make sense to release 0.5.1 with this patch, because SPI2 and SPI3 work with incorrect frequency for now.

Agreed, i'll do that tomorrow after also reviewing #150

**Comment by TheZoq2** [comment]
v0.5.1 (and v0.5.2 because I failed at git) have now been released with this change included :)

**Pull request merged**